Rocket.Chat/Controllers/Chat/ChatDataController.swift

```diff
@@ -259,15 +259,26 @@ final class ChatDataController {
         return (indexPaths, removedIndexPaths)
     }
 
+    func needsUpdate(oldMessage: Message, newMessage: Message) -> Bool {
+        return oldMessage.text != newMessage.text ||
+            oldMessage.type != newMessage.type ||
+            oldMessage.mentions.count != newMessage.mentions.count ||
+            oldMessage.channels.count != newMessage.channels.count ||
```

Comment:
Are you fixing a bug here? 🤔

Reply:
Yes, this change is making the message updates the UI after sending a message and receiving mentions/channels. 😃